### PR TITLE
doc. oh-my-posh

### PR DESCRIPTION
**Pull request by mQzLjP**

### CODE REVIEW

File: config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+1. `notepad $PROFILE` (if error, run `New-Item -Path $PROFILE -Type File -Force` or `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope LocalMachine`)
+2. Add the following line in `$PROFILE`, then SAVE:
+
+```powershell
+oh-my-posh init pwsh | Invoke-Expression
+```
+
+3. run `. $PROFILE` to reload (You can ignore the error message, it might mean you have already changed the prompt before.)
+
```
Comment (zzjc1234) [inline config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md:17]: There is a minor issue here. For win11, tje system forbid to run script by default. I will provide you the command later.

**Review by zzjc1234 — CHANGES_REQUESTED** [review]

File: config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md (hunk 1 above)
Comment (zzjc1234) [reply config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md:17]: The command should be `Set-ExecutionPolicy RemoteSigned`

Comment (zzjc1234) [reply config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md:17]: @mQzLjP Please fix this here.

Comment (mQzLjP) [reply config_tutorials/oh-my-posh for Windows/Part 3 - Changing Prompt.md:17]: fixed

**Review by zzjc1234 — APPROVED** [review]

**Comment by zzjc1234** [comment]
Good job @mQzLjP 🎉 🎉 🎉

**Pull request merged**